src/lib.rs

```diff
@@ -36,12 +49,43 @@ pub fn simulate_bundle(
         .and_then(simulation::simulate_bundle)
 }
 
+/// POST /simulate-stateful-new
```

Comment:
```suggestion
/// POST /simulate-stateful
```

Would prefer just a POST without an id to setup your state. Would be more RESTful then.